.github/workflows/cargo-license.yaml

```diff
@@ -4,5 +4,5 @@ jobs:
   cargo-deny:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4
-    - uses: EmbarkStudios/cargo-deny-action@68cd9c5e3e16328a430a37c743167572e3243e7e
+    - uses: actions/checkout@v3
```

Comment:
Please keep hashes and put required version in the comment. It's important for security.

Also why to downgrade checkout?

Reply:
We have plenty of non-pinned actions used in the codebase, even [introduced by the DevOps team itself](https://github.com/matter-labs/zksync-era/commit/599142e8f74228d7a82cc337f965350c1660e8a1#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR32) 🙃 

I would only call this a recommendation, not a rule, until we have automation in place to enforce it.